sedai/README.md

```diff
@@ -1,59 +1,49 @@
-* Autonomously reduce resource latency by 75%
-* Reduce cloud costs by 30% through autonomous optimization
-* Increase team productivity and agility by 3x
+### How it Works
 
-### Core features
+* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.
 
-* **Easy setup:** Sedai's agentless approach integrates with your cloud accounts and observability platforms so you can get up and running in minutes.
-* **Configuration-free:** By inferring your topology and metrics, Sedai leverages them to detect signals based on observed performance.
-* **Autonomous Availability:** Prevent issues and ensure the highest levels of availability with autonomous actions.
-* **Release Intelligence:** Validate release performance in production in real-time with smart scorecards that assess the quality of your deployments and analyze deviations in latency and errors.
-* **Resource Optimization:** Improve customer experience and lower cloud costs on the fly with dynamic allocation that manages your resources based on traffic needs.
-* **Smart SLOs:** In addition to automatically analyzing and defining targets, Sedai manages error budgets with autonomous actions that prevent errors, downtime, and SLO breaches.
+* **Configuration-free:** Easily connects to Datadog API and intelligently identifies, prioritizes and learns metric behavior.
 
-To learn more, see [Sedai Inc.][1]
+* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.
 
 ## Setup
 
-After connecting your cloud provider, monitoring provider, and notification provider accounts, Sedai identifies your topology and metrics and starts to analyze performance.
+In Sedai:
 
-### Installation
+1. Navigate to Settings > Notifications > Add Integration > Datadog Icon
 
-Existing Sedai customers can receive notifications about Sedai activity in Datadog. For more information, see the [Sedai Notifications][6].
+   ![Add Datadog Integration][3]
 
-## Data Collected
+2. Enter a nickname and the API Key for your Datadog account, enable it and test the integration.
 
-### Metrics
+   ![Setup Datadog API Key][4]
 
-This integration does not collect metrics.
+3. Once the test is verified to be working, click Save.
 
-### Events
+   ![Save Working Datadog Integration][5]
 
-This integration does not support events.
+4. Under Settings > Notifications, [select which notifications][2] you want to send to Datadog. 
 
-### Service
+   ![Enable Datadog Notifications][6]
 
-This integration does not collect service checks.
+## Data Collected
 
-## Support
+This integration sends events into Datadog.
 
-Need help? Email [Support][3] is available 24/7.
+## Support
 
-Alternatively, [schedule a 30-minute demo][2] or [contact us][5] for a [free 14-day trial][2].
+For help with this integration, please email [support@sedai.io][1].
```

Comment:
```suggestion
For help with this integration, contact [Datadog support][1].
```

Reply:
Hi @cswatt ,
Can you please confirm for support email, I should use Datadog's support email
 I've referred to other Readme(s) in the repo (Purefa, Postman  for example) and the support section shows the integration providers email, not that of Datadog.